### DISCUSSION TITLE
OEW and PAYLOAD

### DISCUSSION THREAD

**Discussion by Peter-Mary-Jo** [flybywiresim/aircraft#5756] [Help]
Hello, I'm a little bit confused about the actual value of the OEW of A32nx. According to the fly by wire profile in Simbrief OEW=97700 lbs; according PERF/W+B in AOC MENU OEW=90400 lbs. 
Concerning the PAYLOAD on PERF/W+B page: do PAX weight are included in FWD BAG and REAR BAG?

To all devs, many thanks for making the A32nx. I hope they have as fun to develop it as I have to fly it.

All the best

**Comment by Peter-Mary-Jo** [comment]
I got an answer to my question.
1. on AOC menu and PERF/W+B page: FWD BAG+ REAR BAG=NO PAX multiplied by 280 lbs. NO PAX is the Simbrief Pax number in green.
2. The workaround to match the Simbrief dispatch numbers is to add 7400 lbs to the PAYLOAD number

Happy flying

**Comment by tracernz** [comment]
Not sure on the weights due to lbs, but this might interest you: https://github.com/flybywiresim/a32nx/pull/5547. The payload layout is being revamped and a matching SimBrief profile will be created.

**Reply by Peter-Mary-Jo** [reply]
Thank you I'll check #5547

**Reply by Peter-Mary-Jo** [reply]
Actually the weight per pax is 230 lbs (including bag) instead of 280 lbs.
